app/components/channel_item/channel_item.tsx

```diff
@@ -122,7 +122,7 @@ const ChannelListItem = ({
     const styles = getStyleSheet(theme);
 
     // Make it brighter if it's not muted, and highlighted or has unreads
-    const isBright = !isMuted && (myChannel && (myChannel.isUnread || myChannel.mentionsCount > 0));
+    const isBright = myChannel && (myChannel.isUnread || myChannel.mentionsCount > 0);
```

Comment:
Why was `isMuted` removed?

Reply:
We still want to show the bolder/brighter text even if it's at the new opacity; see https://www.figma.com/file/a9w2q3JnVvBwo2IScP8j3K/Phase-1-Design?node-id=4171%3A136108

With mention, and Unread (first two) still have the original styling - it's just an opacity applied.